nvi-rest/src/main/java/no/sikt/nva/nvi/rest/upsert/UpdateNviCandidateStatusHandler.java

```diff
@@ -57,6 +59,12 @@ protected Integer getSuccessStatusCode(NviStatusRequest input, CandidateResponse
         return HTTP_OK;
     }
 
+    private void verifyHasReasonWhenRejecting(NviStatusRequest input) throws BadRequestException {
+        if (input.status() == NviApprovalStatus.REJECTED && input.reason() == null) {
+            throw new BadRequestException("Missing reason for rejection");
+        }
```

Comment:
Shouldn't it be:
`input.reason() != null` 
Use `nonNull(input.reason())` instead of `!=`